src/main/scala/com/github/t3hnar/bcrypt/package.scala

```diff
@@ -10,11 +12,11 @@ package object bcrypt {
   implicit class Password(val pswrd: String) extends AnyVal {
     def bcrypt: String = B.hashpw(pswrd, BCrypt.gensalt())
 
-    def bcrypt(rounds: Int): String = B.hashpw(pswrd, BCrypt.gensalt(rounds))
+    def bcrypt(rounds: Int): Try[String] = Try(B.hashpw(pswrd, BCrypt.gensalt(rounds)))
```

Comment:
could you please add different methods like `bcryptSafe` to preserve backward compatibility
And also add docs on why it is dangerous to use the old ones `bcrypt`